addons/account/views/report_invoice.xml

```diff
@@ -250,21 +250,6 @@
                         <p t-if="o.invoice_payment_term_id" name="payment_term">
                             <span t-field="o.invoice_payment_term_id.note"/>
                         </p>
```

Comment:
These lines should also be removed i guess

Reply:
Thanks for pointing it, this too seems duplicated.
updated the pr